docs/installations/installation-on-gnu-linux.md

```diff
@@ -0,0 +1,106 @@
+Installation on GNU/Linux
+=========================
+
+## Install via Conda
+
+Use conda or bioconda to install OpenMS.
+
+1. Follow the instructions to [install conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/macos.html).
+2. Install OpenMS using conda:
+   `conda install -c openms openms`
+3. Other OpenMS packages can be installed using:
+   ```
+   conda install -c openms pyopenms
+   conda install -c openms openms-thirdparty
+   conda install -c openms libopenms
+   ```
+
+To install using bioconda:
+
+```
+conda install -c bioconda openms
+conda install -c bioconda/label/cf201901 openms
+conda install -c bioconda openms-tools
```

Comment:
the openms-tools package looks like a very outdated and specific version of the openms package. I think it should be removed.